style-dictionary/visual-refresh/colors.ts

```diff
@@ -143,8 +145,8 @@ const tokens: StyleDictionary.ColorsDictionary = {
   colorStrokeCodeEditorGutterActiveLineDefault: { light: '{colorGrey300}', dark: '{colorGrey750}' },
   colorStrokeCodeEditorGutterActiveLineHover: { light: '{colorGrey100}', dark: '{colorGrey900}' },
   colorTextAccent: { light: '{colorBlue600}', dark: '{colorBlue500}' },
-  colorTextBodyDefault: { light: '{colorGrey900}', dark: '{colorGrey300}' },
-  colorTextBodySecondary: { light: '{colorGrey600}', dark: '{colorGrey300}' },
```

Comment:
We need to add these definitions to `style-dictionary/classic/colors.ts` definition, otherwise the new values will update in classic as well. Previously they were the same between classic and VR, so they were just being inherited.